examples/nextjs/src/app/flow/page.tsx

```diff
@@ -1,14 +1,31 @@
 import { fetchPersonas, FlowProvider } from '@speechmatics/flow-client-react';
-import Component from './Component';
 import { PcmAudioRecorderProvider } from '@speechmatics/browser-audio-input-react';
+import { Controls } from './Controls';
+import { Status } from './Status';
+import { OutputView } from './OutputView';
+import type { Metadata } from 'next';
+
+export const metadata: Metadata = {
+  title: 'Speechmatics Flow example',
+  description: 'Generated by create next app',
+};
 
 export default async function Home() {
   const personas = await fetchPersonas();
 
   return (
     <PcmAudioRecorderProvider workletScriptURL="/js/pcm-audio-worklet.min.js">
```

Comment:
Could we add a comment for what this is doing?

Reply:
Sure, maybe just a link to the README for that package?
https://github.com/speechmatics/speechmatics-js-sdk/blob/main/packages/browser-audio-input-react/README.md

Reply:
Sounds good :)